### PR TITLE
Files from db celery chain

### PR DESCRIPTION
**Pull request by matthiasschaub**
blocker is #194

### CODE REVIEW

File: sketch_map_tool/tasks.py [hunk 1]
```diff
@@ -138,23 +145,25 @@ def c_digitize(color: str, name: str) -> chain:
...
             | group([c_digitize(color, name) for color in COLORS])
             | t_merge.s()
         )
 
-    def c_workflow(files) -> chain:
+    def c_workflow(file_ids: list[int]) -> chain:
         """Start processing workflow for each file."""
         return (
-                group([c_process(file, name) for (file, name) in files])
+                # TODO: replace mock by actual file name
```
Comment (matthiasschaub) [inline sketch_map_tool/tasks.py:161]: TODO

File: sketch_map_tool/database/client.py [hunk 2]
```diff
@@ -11,6 +11,18 @@
 db_conn: connection | None = None
```
Comment (matthiasschaub) [inline sketch_map_tool/database/client.py:13]: Add comment why this is necessary

File: sketch_map_tool/tasks.py [hunk 3]
```diff
@@ -109,20 +111,25 @@ def generate_digitized_results(files: list[tuple[BytesIO, str]]) -> str:
 # group -> group of tasks (parallel)
 #
 # fmt: off
-def georeference_sketch_maps(files, map_frame, bbox) -> str:
+def georeference_sketch_maps(file_ids, map_frame, bbox) -> str:
 
-    def c_process(sketch_map: BytesIO) -> chain:
+    def c_process(sketch_map: int) -> chain:
```
Comment (joker234) [inline sketch_map_tool/tasks.py:116]: maybe `sketch_map_id`?

File: sketch_map_tool/tasks.py [hunk 4]
```diff
@@ -138,23 +145,25 @@ def c_digitize(color: str, name: str) -> chain:
                 | t_enrich.s({"color": color, "name": name})
                 )
 
-    def c_process(sketch_map: BytesIO, name: str) -> chain:
+    def c_process(sketch_map: int, name: str) -> chain:
```
Comment (joker234) [inline sketch_map_tool/tasks.py:148]: see above

File: sketch_map_tool/tasks.py [hunk 5]
```diff
@@ -138,23 +145,25 @@ def c_digitize(color: str, name: str) -> chain:
...
             | t_merge.s()
         )
 
-    def c_workflow(files) -> chain:
+    def c_workflow(file_ids: list[int]) -> chain:
         """Start processing workflow for each file."""
         return (
-                group([c_process(file, name) for (file, name) in files])
+                # TODO: replace mock by actual file name
+                group([c_process(i, "mock") for i in file_ids])
```
Comment (joker234) [inline sketch_map_tool/tasks.py:162]: fix TODO before merge/approve

File: sketch_map_tool/tasks.py (hunk 3 above)
Comment (matthiasschaub) [reply sketch_map_tool/tasks.py:116]: done

**Review by joker234 — APPROVED** [review]

**Review by joker234 — APPROVED** [review]

**Pull request merged**